clang/test/SemaHLSL/BuiltIns/dot-warning.ll

```diff
@@ -0,0 +1,49 @@
+; ModuleID = 'D:\projects\llvm-project\clang\test\SemaHLSL\BuiltIns\dot-warning.hlsl'
```

Comment:
Maybe naive question...but isn't this about the `dot` intrinsic?

Reply:
this file shouldn't be in this pr. this was a mistake.